content/en/docs/instrumentation/python/automatic/agent-config.md

```diff
@@ -28,49 +28,30 @@ opentelemetry-instrument \
 
 Here's an explanation of what each configuration does:
 
-- `traces_exporter` specifies which trace exporter to use. In this case, traces
-  are being exported to `console` (stdout) and to `otlp`. The `otlp` option
-  tells `opentelemetry-instrument` to send it to an endpoint that accepts OTLP
-  via gRPC. The full list of available options for traces_exporter can be found
+- `traces_exporter` specifies which traces exporter to use. In this case, traces
+  are being exported to `console` (stdout) and with `otlp`. The `otlp` option
+  tells `opentelemetry-instrument` to send the traces to an endpoint that accepts OTLP
+  via gRPC. In order to use HTTP instead of gRPC, add `--exporter_otlp_protocol http`.
+  The full list of available options for traces_exporter can be found
   [here](https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/opentelemetry-instrumentation).
-- `otlp` used above for `traces_exporter` is the equivalent of using
-  `otlp_proto_grpc`. To send traces via HTTP instead of gRPC, replace
-  `otlp_proto_grpc` (or `otlp`) with `otlp_proto_http`.
 - `metrics_exporter` specifies which metrics exporter to use. In this case,
   metrics are being exported to `console` (stdout). It is currently required for
   your to specify a metrics exporter. If you aren't exporting metrics, specify
   `none` as the value instead.
-- `service_name` sets the name of the service associated to the trace, and is
-  sent to your [Observability back-end](/ecosystem/vendors/).
-- `exporter_otlp_endpoint` tells `opentelemetry-instrument` to send the traces
-  to the given [Observability back-end's](/ecosystem/vendors/) endpiont via
-  gRPC, or directly to the [OpenTelemetry Collector](/docs/collector/).
+- `service_name` sets the name of the service associated to the telemetries, and is
+  sent to your [Observability backend](/ecosystem/vendors/).
+- `exporter_otlp_endpoint` sets the endpoint which the telemetries are exported to.
```

Comment:
```suggestion
- `exporter_otlp_endpoint` sets the endpoint where telemetry is exported to.
```
Note that "telemetries" is incorrect, as "telemetry" refers to instrumentation data already (and can be plural).